src/components/confirm-appointment/TasksList.tsx

```diff
@@ -0,0 +1,224 @@
+import CloseIcon from '@mui/icons-material/Close';
+import { TextField } from '@mui/material';
+import { useState } from 'react';
+import {
+  ButtonWrapper,
+  StyledButton,
+} from 'src/components/profile/profile-qualification/certificate-list/styles';
+import { FilledButton } from 'src/components/reusable';
+import Modal from 'src/components/reusable/modal/Modal';
+import { useLocales } from 'src/locales';
+import CreditCardIcon from 'src/assets/icons/CreditCardIcon';
+import CheckIcon from 'src/assets/icons/Check';
+import EditIcon from 'src/assets/icons/EditIcon';
+import ConfirmModal from './Modal';
```

Comment:
relative import

Reply:
I think if they are in the same folder it's fine especially if we are talking about styles, types and constants. As for component, will fix)